### PR TITLE
Feature/navigate

### PR DESCRIPTION
**Pull request by andreGarvin**
@rdepena @datamadic

### CODE REVIEW

File: src/api/window/window.ts [hunk 1]
```diff
@@ -476,14 +476,14 @@ export class _Window extends Base {
...
 
     /**
-     * Stops any current navigation the window is performing.
+     * Navigates the window to a specified URL.
+     * @param {string} url - The URL to navigate the window to.
      * @return {Promise.<void>}
-     * @tutorial Window.stopNavigation
+     * @tutorial Window.navigate
      */
-     public stopNavigation(): Promise<void> {
```
Comment (rdepena) [inline src/api/window/window.ts]: @andreGarvin was it your intention to delete stopNavigate.

Comment (andreGarvin) [reply src/api/window/window.ts]: Yes

Comment (rdepena) [reply src/api/window/window.ts]: wait, why ?

**Comment by andreGarvin** [comment]
I thought you merged naviageite from the other PR. But might result to a
merge conflict ethier I am guesssing. Plus stop navigation is in another PR
that dealt with navigation so I ncluded in that one and delete that others

**Comment by rdepena** [comment]
Closing because of #48

**Pull request closed without merging**